packages/components/components/elvis-segmented-control/src/react/styledComponents.tsx

```diff
@@ -66,20 +66,22 @@ export const SegmentedControlContainer = styled.div<SegmentedControlContainerPro
   background: ${({ $type }) => ($type === 'text' ? getThemeColor('background-element-1') : 'transparent')};
 
   // Selected control background
-  ${({ $type, numberOfControls, selectedIndex }) =>
+  ${({ $type, selectedLeft, selectedWidth }) =>
     $type === 'text' &&
     css`
       &::after {
         will-change: left;
```

Comment:
Not related to this PR, but we should consider not animating `left`, since it has a performance impact. We should instead use `translate`.

Reply:
I also noticed we have a `will-change` on the tabs themselves. We should avoid use of `will-change` to not affect performance. 
From MDN:

> Warning: will-change is intended to be used as a last resort, in order to try to deal with existing performance problems. It should not be used to anticipate performance problems.